modules/aws_base/tf_module/log_bucket.tf

```diff
@@ -13,6 +13,16 @@ resource "aws_s3_bucket" "log_bucket" {
   }
 }
 
+resource "aws_s3_bucket_public_access_block" "log_bucket" {
+  bucket = aws_s3_bucket.log_bucket.id
+
+  block_public_acls   = true
+  block_public_policy = true
+  restrict_public_buckets = true
+  ignore_public_acls = true
```

Comment:
Can you run `terraform fmt` on this file? These lines aren't aligned.

Reply:
Done and pushed